client/apps/style.scss

```diff
@@ -196,7 +196,7 @@ a:hover {
     flex-shrink: 0;
 }
 
-.subnav {
+.subnav.subnav--light {
```

Comment:
could you improve naming? The styles here aren't making the subnav light. It'd be better to add an extra classname for this and name it according to the function of these styles.

Reply:
yep